eng/packages/http-client-csharp/generator/TestProjects/Local/Basic-TypeSpec/src/Generated/BasicTypeSpecClient.cs

```diff
@@ -50,11 +52,11 @@ public BasicTypeSpecClient(Uri endpoint, ApiKeyCredential keyCredential, BasicTy
 
             _endpoint = endpoint;
             _keyCredential = keyCredential;
-            Pipeline = ClientPipeline.Create(options, Array.Empty<PipelinePolicy>(), new PipelinePolicy[] { ApiKeyAuthenticationPolicy.CreateHeaderApiKeyPolicy(_keyCredential, AuthorizationHeader) }, Array.Empty<PipelinePolicy>());
+            Pipeline = HttpPipelineBuilder.Build(options, new HttpPipelinePolicy[] {  });
         }
 
         /// <summary> The HTTP pipeline for sending and receiving REST requests and responses. </summary>
-        public ClientPipeline Pipeline { get; }
+        public HttpPipeline Pipeline { get; }
```

Comment:
@jsquire 
You can find the generated SDK for Azure under this test project.
For instance, the Pipeline type has been changed from `ClientPipeline` to `HttpPipeline`.
And the response type of operation has been changed from `ClientResult` to `Response`, and such on.